File: 프로그래머스/1/12912. 두 정수 사이의 합/두 정수 사이의 합.java
class Solution {
    public long solution(int a, int b) {
        // a와 b 중 작은 값과 큰 값을 구함
        int start = Math.min(a, b);
        int end = Math.max(a, b);
        
        // start부터 end까지 모든 수의 합을 계산
        long sum = 0;
        for (int i = start; i <= end; i++) {
            sum += i;
        }
        
        return sum;
    }
}

File: 프로그래머스/1/12912. 두 정수 사이의 합/두 정수 사이의 합.kt
class Solution {
    fun solution(a: Int, b: Int): Long {
        val start = minOf(a, b)
        val end = maxOf(a, b)
        var sum: Long = 0

        for (i in start..end) {
            sum += i
        }

        return sum
    }
}
